packages/components/organisms/f-footer/src/components/LinkList.vue

```diff
@@ -2,7 +2,7 @@
     <div
         v-if="linkList.links.length"
         :class="[$style['c-footer-panel'], {
-            'is-collapsed': panelCollapsed && isBelowWide
+            'is-collapsed': panelMobileCollapsed
```

Comment:
Can we leave a check for `isBelowWide` here for not having this class for wider view?

Reply:
That was causing the issue, as the value changes on scroll on mobile, although those should only be on resize or when mounted. Additionally it seems to add complexity that is does not seem to be needed.

Reply:
Also the check based on `panelCollapsed` only triggers when below wide.

Reply:
So was the bug that `isBelowWide` was changing when the page was scrolled? Just trying to get my head around what was the cause of the issue itself 😄

Reply:
For some reason and only on mobile `isBelowWide` gets a value reset on scroll. I can only explain it with page scrollbar appear when the user scrolls... that has an effect on the page width, and triggers the resize, followed by values reset. 

This change removes the resize collapse reset trigger. The display behaviour is based only on CSS media queries and presence of `is-collapsed`. The resize listener only enables/disables the ability to toggle.

Reply:
makes sense – and agree, it's probably the scrollbar appearing/disappearing triggering it (I think I've come across a similar thing before now you've mentioned that)

Reply:
Correction, It is the collapsing address bar. - https://developers.google.com/web/updates/2016/12/url-bar-resizing 
@ashleynolan ^^